<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Desta" ignored="0" name="Default test" tests="6" failures="1" timestamp="2022-08-06T02:52:03 ICT" time="92.201" errors="0">
  <testcase name="runScenario" time="18.639" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="12.259" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="17.317" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="12.15" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
  <testcase name="runScenario" time="13.336" classname="io.cucumber.testng.AbstractTestNGCucumberTests">
    <failure type="java.lang.AssertionError" message="expected [0] but found [8]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [8]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:839)
at org.testng.Assert.assertEquals(Assert.java:849)
at definitionData.UploadData.UserSpv_get_new_data_import_on_the_page(UploadData.java:174)
at âœ½.UserSpv get new data import on the page or website(file:///C:/Users/ASUS/Desktop/Quality%20Assurance/SQA%20Project/SQA-KerjaPraktek/src/test/resources/DataFeatureFiles/uploadData.feature:12)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase name="runScenario" time="17.649" classname="io.cucumber.testng.AbstractTestNGCucumberTests"/>
</testsuite> <!-- Default test -->
